.gitignore

```diff
@@ -1,11 +1,11 @@
-notebooks/
 trackio/__pycache__/
 tests/__pycache__/
-tf_test_run/
```

Comment:
`notebooks/` and `tf_test_run/` were in the repo but untracked. I assume the gitignore was wrong so fixed it.